client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -215,6 +215,14 @@ button {
 
 }
 
+.hundred-year-plan.step-route {
+	@include break-small {
+		&.setup {
+			margin-top: -60px;
+		}
+	}
+}
```

Comment:
Can't we put this in the step's CSS?

Reply:
@alshakero moved into the step!

Reply:
Thank you sir. I'm trying to keep Stepper itself free from step and flow logic.

Reply:
You're totally right, my bad!